WordPress/WordPressTest/DataMigratorTests.swift

```diff
@@ -88,16 +86,16 @@ class DataMigratorTests: XCTestCase {
         XCTAssertEqual(migratorError, .localDraftsNotSynced)
     }
 
-    func testExportFailsWhenKeychainThrows() {
-        // Given
-        keychainUtils.shouldThrowError = true
-
-        // When
-        let migratorError = getExportDataMigratorError(migrator)
-
-        // Then
-        XCTAssertEqual(migratorError, .keychainError)
-    }
+//    func testExportFailsWhenKeychainThrows() {
+//        // Given
+//        keychainUtils.shouldThrowError = true
+//
+//        // When
+//        let migratorError = getExportDataMigratorError(migrator)
+//
+//        // Then
+//        XCTAssertEqual(migratorError, .keychainError)
+//    }
```

Comment:
Should we remove the test and the `keychainError` error case from the enum?

Reply:
I delayed removing them since I'm unsure if I'd need the keychain bits for my next PR, but turns out I didn't need them 🙂 .  I've removed the commented test in #19663, but good catch about the `keychainError`! I'll remove it on the other PR too.